test/browser/browser.sh

```diff
@@ -12,11 +12,17 @@ LOGS="$(pwd)/logs"
 mkdir -p "$LOGS"
 chmod a+w "$LOGS"
 
-# install browser; on RHEL, use firefox-nightly as chromium broke CDP on rhel-9
-# Install firefox to pull in all the deps
-dnf -y install firefox
-curl --location 'https://download.mozilla.org/?product=firefox-nightly-latest-ssl&os=linux64&lang=en-US' | tar -C /usr/local/lib/ -xj
-ln -s /usr/local/lib/firefox/firefox /usr/local/bin/
+# install browser; on RHEL/CentOS, use firefox
+if grep -q 'ID=.*rhel' /etc/os-release || grep -q 'ID=.*centos' /etc/os-release; then
+    # Install firefox to pull in all the deps
+    dnf install -y firefox
+    curl --location 'https://download.mozilla.org/?product=firefox-nightly-latest-ssl&os=linux64&lang=en-US' | tar -C /usr/local/lib/ -xj
```

Comment:
There is a COPR from someone - maybe it's cleaner? https://copr.fedorainfracloud.org/coprs/wanzenbug/firefox-nightly/

Reply:
code wise I mean.